doc/install/containers.rst

```diff
@@ -0,0 +1,78 @@
+Ceph Container Images
+=====================
+
+Official Releases
+-----------------
+
+Ceph Container images are available from Docker Hub at::
+
+  https://hub.docker.com/r/ceph
+
+
+ceph/ceph
+^^^^^^^^^
+
+- General purpose Ceph container with all necessary daemons and
+  dependencies installed.
+
++--------------+------------------------------------------------------------+
+| Tag          | Meaning                                                    |
++--------------+------------------------------------------------------------+
+| v$rel        | Latest release in this series (e.g., $rel = 14 = Nautilus) |
+| v$rel.2      | Latest *stable* release in this stable series              |
+| v$rel.2.$dot | A specific release                                         |
+| ...          | ...                                                        |
++--------------+------------------------------------------------------------+
+
+ceph/daemon-base
+^^^^^^^^^^^^^^^^
+
+- General purpose Ceph container with all necessary daemons and
+  dependencies installed.
+- Basically the same as *ceph/ceph*, but with different tags.
+
+:note: This image will soon become an alias to *ceph/ceph*.
+
++-----------------------+-----------------------------------------------------+
+| Tag                   | Meaning                                             |
++-----------------------+-----------------------------------------------------+
+| latest-master         | ???                                                 |
+| latest-master-devel   | Daily build of the master branch                    |
+| latest-$release-devel | Daily build of the $release (e.g., nautilus) branch |
++-----------------------+-----------------------------------------------------+
+
+
+ceph/daemon
+^^^^^^^^^^^
+
+- *ceph/daemon-base* plus a collection of BASH scripts that are used
+  by ceph-nano and ceph-ansible to manage a Ceph cluster.
+
++-----------------------+-----------------------------------------------------+
+| Tag                   | Meaning                                             |
++-----------------------+-----------------------------------------------------+
+| latest-master         | ???                                                 |
+| latest-master-devel   | Daily build of the master branch                    |
+| latest-$release-devel | Daily build of the $release (e.g., nautilus) branch |
++-----------------------+-----------------------------------------------------+
+
+
+Development builds
+------------------
+
+We automatically build container images for development branches in the ceph.git
+and ceph-ci.git repositories and push them to Quay at::
+
+  https://quay.io/organization/cephci
+
+ceph/daemon-base
```

Comment:
@dmick I suggest we switch to ``ceph-ci`` for the org and ``ceph/ceph`` as the base container name at the same time

Reply:
:+1:

Reply:
(btw, the reason I chose cephci was probably because of the warning when I create ceph-ci that it's incompatible with docker versions < 1.9.  I doubt it's an issue, but I'll check docker and podman.)